Dockerfile

```
@@ -32,10 +33,29 @@ RUN aws --version # Verify AWS CLI installation.
 # Explicitly set CA cert to resolve SSL issues with AWS.
 ENV AWS_CA_BUNDLE /aws/dist/botocore/cacert.pem
 
-RUN mkdir -p /application
+RUN mkdir -p /application/content-build
+RUN chown -R vets-website:vets-website /application
 
-WORKDIR /application
+WORKDIR /application/
```

```suggestion
WORKDIR /application/content-build
```

I think if we leave this out the normal docker-compose will not function as people expect. I'm not sure